usaspending_api/database_scripts/matview_generator/matview_sql_generator.py

```diff
@@ -315,15 +312,31 @@ def main(source_file):
 
 
 if __name__ == '__main__':
-    if len(sys.argv) > 1:
-        print('Creating matview SQL using {}'.format(sys.argv[1]))
-        main(sys.argv[1])
+    arg_parser = argparse.ArgumentParser(
+        prog='matview_sql_generator.py',
+        description='Generates all of the necessary SQL files for jenkins scripts')
+    arg_parser.add_argument(
+        '--dest',
+        type=str,
+        default='../matviews/',
+        help='Destination folder for all generated sql files')
+    arg_parser.add_argument(
+        '--file',
+        type=str,
+        default=None,
+        help='filepath to the json file containing the sql description')
+    args = arg_parser.parse_args()
+
+    DEST_FOLDER = args.dest
+    if not os.path.exists(os.path.join(DEST_FOLDER, COMPONENT_DIR)):
+        os.makedirs(os.path.join(DEST_FOLDER, COMPONENT_DIR))
+
+    if args.file is not None:
+        if os.path.isfile(args.file):
+            print('Creating matview SQL using {}'.format(args.file))
+            main(args.file)
     else:
-        ans = input('Would you like to run on all json files in dir? (y/N): ')
```

Comment:
so this no longer confirms we want to run for all files, it will just continue with all if it doesn't see the `--files` argument?

Reply:
Yes